prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -37,17 +37,27 @@ public async Task<bool> PecAssignableAsync(int siteId, string pec)
                 .Where(site => site.Id == siteId)
                 .Select(site => new
                 {
-                    site.CareSettingCode,
-                    site.PEC
+                    site.PEC,
+                    healthAuthorityId = (int?)(site as HealthAuthoritySite).HealthAuthorityOrganizationId
                 })
                 .SingleAsync();
 
-            if (siteDto.CareSettingCode == (int)CareSettingType.HealthAuthority
-                || siteDto.PEC == pec)
+            if (siteDto.PEC == pec)
             {
                 return true;
             }
 
+            if (siteDto.healthAuthorityId.HasValue)
+            {
+                return !await _context.HealthAuthoritySites
```

Comment:
As this code is written a health authority site can be made that has the same PEC code as a Community site, but a community site cannot be made if a HA site has the same PEC code.

So one of these ways are wrong, Marty has asked Sarah to clarify but that has to be changed.

Reply:
Confirmed with Sarah that you cannot have a health authority site that has the same PEC as a community site, so this logic needs to be updated.

Reply:
Fixed, now it checks for any sites in other HAs as well as any community site that has the same PEC